src/main/webapp/resources/js/pages/cart/components/CartSamples.jsx

```diff
@@ -65,17 +68,40 @@ const EmptyCartButton = styled(Button)`
   }
 `;
 
-function CartSamplesComponent({
-  samples,
-  applyFilter,
-  emptyCart,
-  displaySample,
-  removeSample,
-  removeProject,
-}) {
-  const filterSamples = (e) => applyFilter(e.target.value);
+export default function CartSamples({ displaySample }) {
+  const [samples, setSamples] = React.useState([]);
+  const {
+    data: allSamples,
+    isSuccess,
+    isFetching,
+    refetch,
+  } = useGetCartQuery();
+  const [emptyCart] = useEmptyMutation();
+  const [removeSample] = useRemoveSampleMutation();
+  const [removeProject] = useRemoveProjectMutation();
+
+  React.useEffect(() => {
+    setSamples(allSamples);
+  }, [allSamples]);
+
+  const filterSamples = (e) => {
```

Comment:
Change parameter `e` to something more clear

Reply:
Updated in b8d4c029682778d53d9631a7e3892abf3614c3ea